tests/lib/tpp_schema.py

```diff
@@ -56,6 +56,22 @@ def event(code, date, numeric_value=None):
     return Events(CTV3Code=code, ConsultationDate=date, NumericValue=numeric_value)
```

Comment:
I know it will blow the diff up, but maybe `Events` and `event` need to be renamed for clarity and consistency?